contracts/LiquidityBridgeContract.sol

```diff
@@ -9,31 +9,53 @@ import './Bridge.sol';
  */
 contract LiquidityBridgeContract {
 
+    int256 constant DUST = 0;
+
+    uint8 constant UNPROCESSED_QUOTE_CODE = 0;
+    uint8 constant CALL_DONE_CODE = 1;
+    uint8 constant PROCESSED_QUOTE_CODE = 2;
+
+    uint32 constant MAX_INT32 = 2147483647;
+    uint32 constant MAX_UINT32 = 4294967295;
+
+    int16 constant BRIDGE_REFUNDED_USER_ERROR_CODE = -100;
+    int16 constant BRIDGE_REFUNDED_LP_ERROR_CODE = -200;
+    int16 constant BRIDGE_UNPROCESSABLE_TX_NOT_CONTRACT_ERROR_CODE = -300;
+    int16 constant BRIDGE_UNPROCESSABLE_TX_INVALID_SENDER_ERROR_CODE = -301;
+    int16 constant BRIDGE_UNPROCESSABLE_TX_ALREADY_PROCESSED_ERROR_CODE = -302;
+    int16 constant BRIDGE_UNPROCESSABLE_TX_VALIDATIONS_ERROR = -303;
+    int16 constant BRIDGE_UNPROCESSABLE_TX_VALUE_ZERO_ERROR = -304;
+    int16 constant BRIDGE_GENERIC_ERROR = -900;
+
     struct Quote {
+ 
         bytes20 fedBtcAddress;
         address lbcAddress;
         address liquidityProviderRskAddress;
         bytes btcRefundAddress;
         address rskRefundAddress;
         bytes liquidityProviderBtcAddress;
-        uint callFee;
-        uint penaltyFee;
+		// SDL: Versioning field is missing. Because no double-hash was used,
+		// then now the version field would need to go here I think
+        uint64 callFee; // SDL: Can't we re-scale to reduce field size?
+        uint64 penaltyFee; // SDL: Can't we re-scale to reduce field size?
         address contractAddress;
         bytes data;        
-        uint gasLimit;
-        uint nonce;
-        uint value;
-        uint agreementTimestamp;
-        uint timeForDeposit;
-        uint callTime;
-        uint depositConfirmations;
+        uint32 gasLimit; // SDL: Do we really need 2^256 range?
+        uint16 nonce; // SDL: Why not uint32 ?
```

Comment:
Sure it will fit in 16 bits ? :)

Reply:
My mistake. Replaced to int64 to be consistent with the LP